homeassistant/components/qnap/coordinator.py

```diff
@@ -34,14 +34,14 @@ def __init__(self, hass: HomeAssistant, config: ConfigType) -> None:
         """Initialize the qnap coordinator."""
```

Comment:
The type hint here is now wrong.
I suggest that you rename `config: ConfigType` to `config_entry: ConfigEntry`